mlflow/entities/run_info.py

```diff
@@ -20,8 +21,6 @@ class RunInfo(_MLflowObject):
     """
     Metadata about a run.
     """
-    ACTIVE_LIFECYCLE = "active"
```

Comment:
AFAICT the lifecycle stage names are identical for runs & experiments, so seems like a non-breaking change to combine the two into a single enum. I think it also makes sense to do in that even if there end up being other stages specific to a given entity (e.g. a lifecycle stage specific to runs or experiments), we can just maintain a set of valid lifecycle stages per entity & filter there.

Reply:
Yeah -- that's what I was thinking, as well. Also centralizing the logic of mapping view types to lifecycle stage is a huge plus.